va/va.h

```diff
@@ -210,6 +210,8 @@ typedef int VAStatus;	/** Return status type from functions */
 #define VA_STATUS_ERROR_HW_BUSY	                0x00000022
 /** \brief An unsupported memory type was supplied. */
 #define VA_STATUS_ERROR_UNSUPPORTED_MEMORY_TYPE 0x00000024
+/** \brief Indicate allocated buffer size is not enough for input or output. */
```

Comment:
when add a new error type, need to update const char *vaErrorStr(VAStatus error_status) https://github.com/intel/libva/blob/b2fdaf4eb465269656417eb03699ea47d0e532bd/va/va.c#L534 at the same time